doc_source/constructs.md

```diff
@@ -1,104 +1,100 @@
+
+Similarly to the AWS SDKs and AWS CloudFormation, the AWS Construct Library is organized into modules, one for each AWS service. For example, the `@aws-cdk/aws-ec2` module includes resources for EC2 instances, Auto Scaling Groups and VPCs, the aws-sns module includes resources such as Topic and Subscription, and so forth. See the [Reference](https://awslabs.github.io/aws-cdk/reference.html#reference) section for descriptions of the AWS CDK packages and constructs\.
+
+AWS Construct Library members are found in the `@aws-cdk/aws-NAMESPACE` packages, where NAMESPACE is the short name for the associated service, such as SQS for the AWS Construct Library for the Amazon SQS service\.
 
 ## Construct Structure<a name="constructs_structure"></a>
 
-The construct tree structure is a powerful design pattern for composing high\-level abstractions\. For example, you can define a `StorageLayer` construct that represents your application's storage layer and include all the AWS resources, such as DynamoDB tables and Amazon S3 buckets, needed to implement your storage layer in this construct\. When your higher\-level code uses this construct, it only needs to instantiate the `StorageLayer` construct\.
+Constructs are represented as normal classes in your code and are defined by instantiating an object of that class.
 
-When you initialize a construct, add the construct to the construct tree by specifying the parent construct as the first initializer parameter, an identifier for the construct as the second parameter, and a set of properties for the final parameter, as shown in the following example\.
+When constructs are initialized, they are always defined within the _**scope**_ of another construct and they always have an _**id**_ which must be unique within the same scope.
 
-```
-new SomeConstruct(parent, name[, props]);
+For example, here's how you would define an SNS Topic in your stack with default configuration:
+
+```ts
+new sns.Topic(this, 'MyTopic');
 ```
 
-In almost all cases, you should pass the keyword `this` for the `parent` argument, because you will generally initialize new constructs in the context of the parent construct\. Any descriptive string will do for the `name` argument, and an in\-line object for the set of properties\.
+The first argument to every construct is always the scope in which it is created, and will almost always be simply `this` because most constructs will be defined within the current scope.
```

Comment:
We don't need to explain scope. It's a fundamental programming concept.

Reply:
I kind of agree with @sam-goodwin - maybe we could say something like "...defined within the current scope, which is normally the initializer (constructor) or a method of some construct defined in your app".

Something like that...

Reply:
It is a fundamental programming concept, but it is ambiguous in this context since we are just now introducing them to the CDK's implementation of scopes. For example, will a novice understand that the scope is also a construct?